test/e2e/specs/wp-signup-spec.js

```diff
@@ -1647,99 +1648,99 @@ describe( `[${ host }] Sign Up  (${ screenSize }, ${ locale })`, function () {
-			const createYourAccountPage = await CreateYourAccountPage.Expect( driver );
-			return await createYourAccountPage.enterEmailAndSubmit( emailAddress );
-		} );
-
-		step(
-			'Can then see the domains page, and Can search for a blog name, can see and select a free .wordpress address in the results',
-			async function () {
-				const findADomainComponent = await FindADomainComponent.Expect( driver );
-				await findADomainComponent.searchForBlogNameAndWaitForResults( blogName );
-				await findADomainComponent.checkAndRetryForFreeBlogAddresses(
-					expectedBlogAddresses,
-					blogName
-				);
-				const actualAddress = await findADomainComponent.freeBlogAddress();
-				assert(
-					expectedBlogAddresses.indexOf( actualAddress ) > -1,
-					`The displayed free blog address: '${ actualAddress }' was not the expected addresses: '${ expectedBlogAddresses }'`
-				);
-				return await findADomainComponent.selectFreeAddress();
-			}
-		);
-
-		step( 'Can see the plans page and pick the free plan', async function () {
-			const pickAPlanPage = await PickAPlanPage.Expect( driver );
-			return await pickAPlanPage.selectFreePlan();
-		} );
-
-		step(
-			'Can then see the sign up processing page which will finish automatically move along',
-			async function () {
-				return await new SignUpStep( driver ).continueAlong( blogName, passwordForTestAccounts );
-			}
-		);
-
-		sharedSteps.canSeeTheOnboardingChecklist();
-
-		step( 'Can see email containing verification link', async function () {
-			if ( process.env.HORIZON_TESTS === 'true' ) {
-				return this.skip();
-			}
-
-			const emailClient = await new EmailClient( signupInboxId );
-			const validator = ( emails ) =>
-				emails.find( ( email ) => email.subject.includes( emailAddress ) );
-			const emails = await emailClient.pollEmailsByRecipient( emailAddress, validator );
-
-			for ( const email of emails ) {
-				if ( email.subject.includes( emailAddress ) ) {
-					return ( verificationLink = email.html.links[ 0 ].href );
-				}
-			}
-			return assert( verificationLink !== undefined, 'Could not locate the login link email.' );
-		} );
-
-		step( 'Can open verification link and verify account', async function () {
-			if ( process.env.HORIZON_TESTS === 'true' ) {
-				return this.skip();
-			}
-			await driver.get( verificationLink );
-			const myHomePage = await MyHomePage.Expect( this.driver );
-			return await myHomePage.isEmailVerified();
-		} );
-
-		after( 'Can delete our newly created account', async function () {
-			// Get username from Account settings page
-			// (it's automatically generated for passwordless signup)
-			const accountSettingsPage = await AccountSettingsPage.Visit( this.driver );
-			const username = await accountSettingsPage.getUsername();
-
-			return await new DeleteAccountFlow( driver ).deleteAccount( username );
-		} );
-	} );
+	// describe.skip( 'Passwordless signup @parallel', function () {
```

Comment:
Hmm, this is already skipped (note `.skip`) so odd if it was causing troubles?

Reply:
I didn't notice the `.skip` and commented it out because it was using `JetpackAddNewSitePage.Visit( driver )`. I'll add it back in for now.